release/resources/important_dlc_packages.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Leave some packages which might also be taken in list, FYI only:
```
scipy
scikit-learn
pandas
Pillow
h5py
fastai
requests
mxnet-model-server (multi-model-server)
mpi (mpirun --version)
openssl (openssl version)
```

Reply:
Added all packages except for mpi and openssl, these would require changes currently outside the scope of this PR. The mpi versions  and openssl don't change significantly so I'm not sure if they'll qualify for this release. Even so, they could be added in later.